src/pages/Scheduling/components/CreateScheduleExceptionSheet.tsx

```diff
@@ -132,6 +134,15 @@ export default function CreateScheduleExceptionSheet({
     },
   });
 
+  const { data: isSchedulableResource } = useQuery({
+    queryKey: ["practitioners", facilityId],
+    queryFn: query(scheduleApis.appointments.availableUsers, {
+      pathParams: { facility_id: facilityId },
+    }),
+    select: (data) =>
+      (data as { users: UserBase[] }).users.some(({ id }) => userId === id),
+  });
```

Comment:
since this useQuery is duplicated, let's create a new hook `useIsUserSchedulableResource` that returns the useQuery directly with appropriate `queryKey`